lib/postprocessing/image.js

```diff
@@ -634,13 +634,21 @@ async function imagePostProcess(intermediateRendition, rendition, directories) {
 
 function convertSvg(img, instructions) {
     // ImageMagick automatically keeps aspect ratio
-    // Only using width because ImageMagick will use the smallest value whether it be width or height
-    const width = instructions.width || SVG_DEFAULT_WIDTH;
+    let imageSize = SVG_DEFAULT_WIDTH;
+
+    if(instructions.width && instructions.height) {
+        // image magick will keep aspect ratio
+        imageSize = `${instructions.width}x${instructions.height}`;
+    } else if(instructions.width) {
+        imageSize = `${instructions.width}x`;
+    } else if(instructions.height) {
+        imageSize = `x${instructions.height}`;
```

Comment:
The x is left for clarity in the command param, and to make sure aspect ratio is kept, since height is the second param.